packages/admin/cms-admin/src/dam/MoveDamItemDialog/ChooseFolder.tsx

```diff
@@ -0,0 +1,312 @@
+                                    Icon={folderTree.has(folder.id) ? (expandedIds.has(folder.id) ? TreeCollapse : TreeExpand) : undefined}
+                                    onIconClick={() => {
+                                        if (expandedIds.has(folder.id)) {
+                                            setExpandedIds((expandedIds) => {
+                                                const newExpandedIds = new Set(expandedIds);
+                                                newExpandedIds.delete(folder.id);
+                                                return newExpandedIds;
+                                            });
+                                        } else {
+                                            setExpandedIds((expandedIds) => {
+                                                const newExpandedIds = new Set(expandedIds);
+                                                newExpandedIds.add(folder.id);
+                                                return newExpandedIds;
+                                            });
+                                        }
+                                    }}
+                                    message={folderMatches ? <MarkedMatches text={folder.name} matches={folderMatches} /> : folder.name}
+                                    offset={20 + 36 * level}
+                                    isChosen={selectedId === folder.id}
+                                    onClick={() => {
+                                        onFolderClick(folder.id);
+                                    }}
+                                    style={style}
+                                />
+                            );
+                        }}
+                    </List>
+                );
+            }}
+        </AutoSizer>
+    );
+};
+
+const StyledListItem = styled(ListItem)<{ offset: number; isChosen?: boolean }>`
+    display: flex;
+    justify-content: space-between;
+
+    height: 56px;
+    background: white;
+    cursor: pointer;
+    padding-left: ${({ offset }) => offset}px;
+
+    border-left: solid 2px transparent;
+    border-right: solid 2px transparent;
+    border-bottom: solid 1px ${({ theme }) => theme.palette.grey[100]};
+
+    &:last-child {
+        border-bottom: none;
+    }
+
+    &:hover {
+        border-left: solid 2px ${({ theme }) => theme.palette.primary.main};
+        border-right: solid 2px ${({ theme }) => theme.palette.primary.main};
+        background: ${({ theme }) => theme.palette.grey[50]};
+    }
+
+    ${({ isChosen, theme }) => {
+        return isChosen
+            ? `
+                border-top: solid 2px ${theme.palette.primary.main};
+                border-bottom: solid 2px ${theme.palette.primary.main};
+                border-left: solid 2px ${theme.palette.primary.main};
+                border-right: solid 2px ${theme.palette.primary.main};
+                
+                &:last-child {
```

Comment:
Is this necessary?

Reply:
This is necessary when choosing the last item since this item has no border.

https://github.com/vivid-planet/comet/blob/c00298773501c0cfff9d01dab3eef8daf3d195e3/packages/admin/cms-admin/src/dam/MoveDamItemDialog/ChooseFolder.tsx#L259-L261

In order to override this

https://github.com/vivid-planet/comet/blob/c00298773501c0cfff9d01dab3eef8daf3d195e3/packages/admin/cms-admin/src/dam/MoveDamItemDialog/ChooseFolder.tsx#L274-L276

is necessary